public/javascripts/EditProduct.js

```diff
@@ -106,19 +109,20 @@ var editProduct = {
     breakAssociations: function() {
         var host = "/removeCustomersByProductID/"
             + window.args.ProductID;
-        navigation.hit(host,function(response){
-            if( response == "Success" ){
+        navigation.get(host, function(err, response){
+            if(err){
+                $("#message").text("Error: " + err.responseText);
+            }
+            else if( response == "Success" ){
                 editProduct.submit();
             } else {
```

Comment:
This was to preserve the flow that was already in place. The else there was testing for a non-failure response where the returned response is not "success", which is an accessable case.